### DISCUSSION TITLE
multipart/form-data

### DISCUSSION THREAD

**Discussion by yeaharh** [microsoft/typespec#2653] [Q&A] [answered]
Hi

Looking to define a Typespec file that will support multipart/form-data content-type for uploading multiple files.

The OpenAPI specification we're looking to replicate in Typespec is attached.

[virus-scan-openapi-spec.json](https://github.com/microsoft/typespec/files/13308198/virus-scan-openapi-spec.json)

Rgds
Jim

**Comment by mikekistler** [comment, marked as answer]
Check out these examples in my typespec-openapi3-examples project:

https://github.com/mikekistler/typespec-openapiv3-examples/blob/main/README.md#multipartform-data

**Reply by yeaharh** [reply]
Thanks Mike
